src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
@mattiaerre so I'm not seeing any falcor errors. here's my network activity and console:

<img width="681" alt="screen shot 2017-03-02 at 1 30 47 pm" src="https://cloud.githubusercontent.com/assets/178183/23527889/88364950-ff4c-11e6-9b29-7dacda84d4af.png">
<img width="681" alt="screen shot 2017-03-02 at 1 31 04 pm" src="https://cloud.githubusercontent.com/assets/178183/23527896/8d45ee0a-ff4c-11e6-921d-c967d283a3f5.png">